### PR TITLE
Add initial syncing, heh

### PR DESCRIPTION
**Pull request by odinuge**
Screenshot from 2021-01-19 19-54-47

#yolo

### CODE REVIEW

**Review by SmithPeder — COMMENTED** [review]
I'm trusting you with the correctness of this script. Two small comments. 
1. I suggest that we move this `sync` folder to `deployment/sync` since it's not directly connected to vote, but rather a specific implementation of a "vote like deployment" used by 
Abakus.
2. Maybe write a short `README.md` that explains some of the tokens so others can find "The token you know" and that kind of stuff hehe :)

**Comment by odinuge** [comment]
Yeah, will fix when the api/env-stuff stabelizes :+1:

File: sync/main.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		for {
+			_, message, err := c.ReadMessage()
+			if err != nil {
+				log.Println("read:", err)
+				return
+			}
+			var abstractAction LegoAction
+			err = json.Unmarshal(message, &abstractAction)
+			if err != nil {
+				log.Printf("Error decding msg: %q with err: %e\n", message, err)
```
Comment (LudvigHz) [inline sync/main.go:111]: `decoding`

File: sync/main.go [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+				log.Printf("Posting to VOTE: %q", string(out))
+				req, _ := http.NewRequest("POST", *voteEndpoint, bytes.NewBuffer(out))
+				req.Header.Add("CSRF-Token", *csrfToken)
+				req.Header.Add("content-type", "application/json;charset=UTF-8")
+				req.Header.Set("Cookie", *authCookieVote)
+
+				resp, err = client.Do(req)
+
+				defer resp.Body.Close()
+				respData, _ = ioutil.ReadAll(resp.Body)
```
Comment (LudvigHz) [inline sync/main.go:158]: Could this give an error?

**Review by LudvigHz — APPROVED** [review]

File: sync/main.go (hunk 2 above)
Comment (odinuge) [reply sync/main.go:158]: Yes and no. We don't handle 100% of errors in lego in general, but will fix error handling when stuff works as expected

**Review by SmithPeder — APPROVED** [review]
Nice

**Pull request merged**